scss/layouts/maternity-care-sharing-intro.scss

```diff
@@ -0,0 +1,72 @@
+$breakpoints: (
+  'xs-phone': 320px,
+  'phone': 480px,
+  'tablet': 768px,
+  'desktop': 1024px,
+  'widescreen': 1200px
+);
+
+// keywords
+$media-expressions: (
+  'screen': 'screen',
+  'print': 'print',
+  'handheld': 'handheld',
+  'landscape': '(orientation: landscape)',
+  'portrait': '(orientation: portrait)',
+  'retina2x': '(-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 144dpi)',
+  'retina3x': '(-webkit-min-device-pixel-ratio: 2.5), (min-resolution: 240dpi)'
+);
+
+@import '../vendors/include-media';
+@import '../base/mixins';
+@import '../base/variables';
+@import '../base/helpers';
+
+
+.maternity-care-sharing-intro__container {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 650px;
+}
+
+.maternity-care-sharing-intro__header {
```

Comment:
Heading font should shrink at smaller sizes.
See https://github.com/pebcakerror/layouts/blob/7d9c244a26fa984c7e4fbf10edb8699c205545ab/scss/layouts/healthcare-confusion-intro.scss#L44

Compare this font to the XD file, I think the weight needs to be a little lighter.